include/geojson/features/FeatureBase.hpp

```diff
@@ -210,6 +210,12 @@ namespace geojson {
              * @return The property identified by the key
              */
             virtual JSONProperty get_property(std::string key) const {
+                if (properties.find(key) == properties.end()) {
+                    std::string key_str(key);
```

Comment:
This is an unnecessary copy of the key.  Its already a string, just add it to the error_message as ` + key + `

Reply:
Removed. Thanks.